nautobot/core/graphql/base.py

```diff
@@ -0,0 +1,10 @@
+from graphene_django import DjangoObjectType
+
+
+class NautobotObjectType(DjangoObjectType):
```

Comment:
Just a note that we need to update the sample code in `examples/dummy_plugin/dummy_plugin/graphql/types.py` and the docs in `nautobot/docs/plugins/development.md` to make use of this new class and document why it's required.